app/Product.php

```diff
@@ -7,4 +7,7 @@
 class Product extends Model
 {
     //
+    public function category(){
```

Comment:
untuk relasi **one-to-many** silahkan lihat di https://laravel.com/docs/5.3/eloquent-relationships#one-to-many , pada hal ini bisa menggunakan `belongsTo` sebagai inverse dari `hasMany` cc @devinlukianto @kharisisriyant